src/SmartComponents/Advisor/Advisor.js

```diff
@@ -1,29 +1,59 @@
 
-import { INCIDENT_URL, NEW_REC_URL } from './Constants';
+import { Button, Divider, Title, TitleSizes, Tooltip, TooltipPosition } from '@patternfly/react-core/dist/esm/components';
+import { Flex, FlexItem, Grid, GridItem } from '@patternfly/react-core/dist/esm/layouts';
 import React, { useEffect, useState } from 'react';
 import { TemplateCard, TemplateCardBody, TemplateCardHeader } from '../../PresentationalComponents/Template/TemplateCard';
+import {
+    global_palette_cyan_100,
+    global_palette_cyan_200,
+    global_palette_cyan_300,
+    global_palette_cyan_400
+} from '@patternfly/react-tokens/dist/esm/';
 
-import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
 import FailState from '../../PresentationalComponents/FailState/FailState';
+import { INCIDENT_URL } from './Constants';
+import InfoIcon from '../../Icons/InfoIcon';
+import { InsightsLabel } from '@redhat-cloud-services/frontend-components/components/esm/InsightsLabel';
 import Loading from '../../PresentationalComponents/Loading/Loading';
-import { NumberDescription } from '../../PresentationalComponents/NumberDescription/NumberDescription';
+import { PieChart } from '../../ChartTemplates/PieChart/PieChartTemplate';
 import PropTypes from 'prop-types';
 import { SEVERITY_MAP } from './Constants';
-import StackChartTemplate from '../../ChartTemplates/StackChart/StackChartTemplate';
 import { UI_BASE } from '../../AppConstants';
 import { connect } from 'react-redux';
 import messages from '../../Messages';
 import { useIntl } from 'react-intl';
 
-/**
- * Advisor Card for showing count/severity of rec hits
- */
 const Advisor = ({ recStats, recStatsStatus, advisorFetchStatsRecs, advisorFetchStatsSystems,
-    advisorIncidents, advisorIncidentsStatus, advisorFetchIncidents, systemsStats, systemsStatsStatus, selectedTags }) => {
+    advisorIncidents, advisorIncidentsStatus, advisorFetchIncidents, selectedTags }) => {
 
     const intl = useIntl();
-    const [chartData, setChartData] = useState([]);
+    const [trData, setTRData] = useState([]);
+    const [categoryData, setCategoryData] = useState([]);
+
+    const colorScale = [
+        global_palette_cyan_100.value,
+        global_palette_cyan_200.value,
+        global_palette_cyan_300.value,
+        global_palette_cyan_400.value
+    ];
+
+    const pieLegendData = categoryData.map(item => ({ name: `${item.y} ${item.x}`, symbol: { fill: `${item.fill}`, type: 'square' } }));
+    const pieLegendClick = categoryData.map(({ value }) => `${UI_BASE}/advisor/recommendations?category=${value}&reports_shown=true&impacting=true&offset=0&limit=10${selectedTags.length && '&tags=' + selectedTags.join()}`);
+    const totalRiskUrl = (risk) => `${UI_BASE}/advisor/recommendations?total_risk=${risk}&reports_shown=true&impacting=true&offset=0&limit=10${selectedTags.length && '&tags=' + selectedTags.join()}`;
+
+    const iconTooltip = text => <Tooltip
+        key={ `${text}` }
```

Comment:
No need for string interpolation
```suggestion
        key={text}
```